CHANGELOG.md

```diff
@@ -25,6 +25,9 @@ All notable changes to this project will be documented in this file based on the
 
 ### Improvements
 * Improved the definition of the file fields #196
+Improved the definition of the agent fields two places #192
+* Improved the definition of the agent fields two places #192
```

Comment:
Seems like we had a `git rebase` issue here :-)

Just edit the file, commit again and push again (no need for the `--force` this time)